envsec/pkg/envsec/set.go

```diff
@@ -0,0 +1,113 @@
+package envsec
+
+import (
+	"context"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/samber/lo"
+	"go.jetpack.io/envsec/internal/tux"
+)
+
+func (e *Envsec) Set(ctx context.Context, envID EnvID, name string, value string) error {
+	return e.SetMap(ctx, envID, map[string]string{name: value})
+}
+
+func (e *Envsec) SetMap(ctx context.Context, envID EnvID, envMap map[string]string) error {
+	err := ensureValidNames(lo.Keys(envMap))
+	if err != nil {
+		return errors.WithStack(err)
+	}
+
+	err = e.store.SetAll(ctx, envID, envMap)
+	if err != nil {
+		return errors.WithStack(err)
+	}
+	insertedNames := lo.Keys(envMap)
+	return tux.WriteHeader(e.Stderr,
+		"[DONE] Set environment %s %v in environment: %s\n",
+		tux.Plural(insertedNames, "variable", "variables"),
+		strings.Join(tux.QuotedTerms(insertedNames), ", "),
+		strings.ToLower(envID.EnvName),
+	)
```

Comment:
I don't know if it makes sense to have the package write Info messages like this to stderr. 

The reason is that caller applications like Devbox may have their own libraries (like Devbox's `ux.Finfo`) for emitting information like this.

Don't want to block on this but could iterate on in the future

Reply:
This is already doable as is, if devbox wants to emit its own messages it can simply pass dev null or a buffer as Stderr. This would silence all output. This way we don't need each user to the library to implement its own printing, they only do it if they want.